integration/sql/README.md

```diff
@@ -2,15 +2,15 @@
 
 Library that extracts lineage from sql statements. 
 
-### Usage
+## Usage
 
 If you're using OpenLineage integration, there's good chance that you're already using this integration.
 
-This library can be used both as Python library and as Rust library, however it's not published at Cargo yet.
+This library is implemented in Rust and provides a Python and Java interface. The Rust implementation has not yet been published to Cargo.
```

Comment:
Wouldn't be better to have separate libraries for python and java?

Reply:
Not sure what you mean - they are not one single binary/file that provides both interfaces. This means that the Python and Java interfaces depend on the same native code.